Dockerfile.rhel8

```diff
@@ -22,7 +22,7 @@ COPY requirements/requirements.insights.txt /tmp/requirements.txt
 # NOTE: en_US.UTF-8 locale is provided by glibc-langpack-en
 RUN set -ex; \
     DNF="dnf -y --disableplugin=subscription-manager" && \
-    INSTALL_PKGS="glibc-langpack-en git-core libpq python38" && \
+    INSTALL_PKGS="glibc-langpack-en git-core libpq python39" && \
     INSTALL_PKGS_BUILD="gcc libpq-devel python38-devel openldap-devel" && \
```

Comment:
@newswangerd quay did not build an image for this PR (which is not usually a problem), im trying again but maybe this `python38-devel` is the cause?

Reply:
lmk if it builds now